### ISSUE TITLE
[BetterMessageLinks] Discord crash

### ISSUE THREAD

**Issue by XT8SS** [TheGreenPig/BetterDiscordPlugins#14]
Clicking on any server or DM while having the plugin enabled will crash Discord and leave you in an infinite loop if you try to reload (since Discord leaves you in the same channel you clicked on).

**Comment by estroBiohacker** [comment]
i've been having the same issue too.

**Comment by Seath1337** [comment]
Same here as well!

**Comment by RectangularObject** [comment]
CrashRecovery gave me this message after the crash
```log
1797be3c832a2db555ad.js:130 [CrashRecovery] HEY OVER HERE! Show this to the plugin developer or in the support server!
Prettified stacktraces, stack:
 Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. 
Component stack:
     at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:3458:857)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:3522:540)
    at BetterLink (betterdiscord://plugins/BetterMessageLinks.plugin.js:274:4)
    at div
    at MessageContent
    at div
    at div
    at t.default (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2186:9)
    at li
    at t.default (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2186:9)
    at betterdiscord://plugins/0PluginLibrary.plugin.js:1650:73
    at ol
    at div
    at T (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2197:293)
    at div
    at https://discord.com/assets/c98c01dfb03a53be67b9.js:1798:624
    at div
    at h (https://discord.com/assets/8ad107d4da9ca914b9f8.js:4069:696)
    at K (https://discord.com/assets/c98c01dfb03a53be67b9.js:1188:11)
    at main
    at De (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:83:861)
    at https://discord.com/assets/c98c01dfb03a53be67b9.js:1810:543
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:3655:407)
    at ChatLayerProvider
    at t (https://discord.com/assets/c7c24887100fd214f8e4.js:30:689)
    at t (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:91:780)
    at https://discord.com/assets/9f5ff8f67c6b94a0f472.js:97:172
    at div
    at div
    at t (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:61:272)
    at e (https://discord.com/assets/4b582ee1842099625350.js:50:233)
    at https://discord.com/assets/c7c24887100fd214f8e4.js:67:764
    at https://discord.com/assets/9f5ff8f67c6b94a0f472.js:75:282
    at pe (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:42:744)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6114:926)
    at c (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2018:11)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6117:10)
    at div
    at div
    at div
    at t.default (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:36:22)
    at div
    at l (https://discord.com/assets/8ad107d4da9ca914b9f8.js:4715:10)
    at t (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:145:351)
    at div
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:4616:162)
    at t (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:147:946)
    at Y (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:149:466)
    at div
    at t (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:308:360)
    at j (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:125:188)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6114:926)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6117:10)
    at t (https://discord.com/assets/9f5ff8f67c6b94a0f472.js:229:369)
    at Suspense
    at d
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6114:926)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6117:10)
    at div
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:3655:407)
    at VerificationLayerProvider
    at E (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2501:293)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:3655:407)
    at AppLayerProvider
    at div
    at div
    at E (https://discord.com/assets/8ad107d4da9ca914b9f8.js:1942:481)
    at T (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2197:293)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:745:655)
    at m (https://discord.com/assets/8ad107d4da9ca914b9f8.js:749:113)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:749:584)
    at u (https://discord.com/assets/8ad107d4da9ca914b9f8.js:4270:738)
    at h (https://discord.com/assets/8ad107d4da9ca914b9f8.js:729:558)
    at re (https://discord.com/assets/8ad107d4da9ca914b9f8.js:733:313)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:790:101)
    at u (https://discord.com/assets/8ad107d4da9ca914b9f8.js:4269:679)
    at https://discord.com/assets/8ad107d4da9ca914b9f8.js:4780:30
    at H (https://discord.com/assets/8ad107d4da9ca914b9f8.js:5928:700)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6111:570)
    at t (https://discord.com/assets/8ad107d4da9ca914b9f8.js:6067:817)
    at _ (https://discord.com/assets/8ad107d4da9ca914b9f8.js:2548:681)
(anonymous) @ 1797be3c832a2db555ad.js:130
```
For everyone else coming here about the crash, get CrashRecovery and move BetterMessageLinks out of your plugins folder temporarily to disable it.

**Comment by j-px** [comment]
Having the same issue. Temporarily disabled the plugin so that my Discord client even works.

**Comment by skunkmommy** [comment]
Yeah, this plugin is fucked.

**Comment by TheGreenPig** [comment]
Fixed in `1.4.12`.
For the future: Please be a little more patient, I have a life too, and it hasn't even been a day. 
Adding comments like “Same” or other bothersome comments don't really help. If you want to provide actual useful information, like what discord version you're using, crash logs or similar, I am grateful for that. 
If you are experiencing the same issue as the author of the issue, you can like the original comment.
